Build/libHttpClient.141.Android.C/libHttpClient.141.Android.C.vcxproj

```diff
@@ -131,8 +131,9 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\Include\XAsync.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\Include\XAsyncProvider.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\Include\XTaskQueue.h" />
+    <ClInclude Include="..\..\Include\httpClient\async_jvm.h" />
```

Comment:
>="..\..\ [](start = 22, length = 8)

MSBuildThisFileDirectory?

Reply:
Should you be updating the CMake stuff?

---
In reply to: [262319532](https://github.com/Microsoft/libHttpClient/pull/387#discussion_r262319532) [](ancestors = 262319532)

Reply:
yes